tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -89,7 +89,7 @@ func NewSetStoreWeightCommand() *cobra.Command {
 // NewStoreLimitCommand returns a limit subcommand of storeCmd.
 func NewStoreLimitCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "limit [<type>]|[<store_id>|<all> <limit> <type>]",
+		Use:   "limit [<type>]|[<store_id>|<all> [<key> <value>]... <limit> <type>]",
```

Comment:
What mean of key/value?

Reply:
They are just like `store label` command.